include/powsybl/stdcxx/reference_wrapper.hpp

```diff
@@ -33,6 +33,11 @@ class reference_wrapper {
 
     reference_wrapper(reference_wrapper&&) noexcept = default;
 
+    reference_wrapper& operator=(T& reference) {
```

Comment:
A const version may be usefull also, what do you think ?

Reply:
The `const` qualifier is carried by the T. You should not be able to assign a `const T&` to a `Reference<T>`. This should be possible only with `CReference<T>`. This is the reason why the T is not const.